packages/zendesk-adapter/src/filters/unordered_lists.ts

```diff
@@ -227,6 +227,17 @@ const sortChildFields = (formInstances: InstanceElement[], ticketFieldById: Reco
   })
 }
 
+const sortRestrictedBrands = (formInstances: InstanceElement[]): void => {
+  formInstances.forEach(form => {
+    const restrictedBrands = form.value.restricted_brand_ids
+    if (_.isArray(restrictedBrands) && restrictedBrands.every(isReferenceExpression)) {
+      form.value.restricted_brand_ids = _.sortBy(restrictedBrands, brand => brand.elemID.getFullName())
+    } else {
+      log.warn(`could not sort restricted brands for ${form.elemID.getFullName()}`)
```

Comment:
This is very unlikely to reach here, but if we do it'd be hard to debug. Let's add a trace on the actual values
```suggestion
      log.warn(`could not sort restricted brands for ${form.elemID.getFullName()}`)
      log.trace(`restricted brands are: %s`, inspectValue(restrictedBrands, { maxArrayLength: null }))
```